src/Utils/utils.ts

```diff
@@ -544,3 +544,13 @@ export const fahrenheitToCelsius = (fahrenheit: number) => {
 export const keysOf = <T extends object>(obj: T) => {
   return Object.keys(obj) as (keyof T)[];
 };
+
+export const handleNegativeValue = (
```

Comment:
`handleNegativeValue` does not convey what the function does. Like what does it handle exactly? Fn. name can be improved.

```suggestion
export const sanitizePositiveNumberValue = (
```